bin/lint.sh

```diff
@@ -58,7 +58,7 @@ IFS=$'\n\t'
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 if $HELP ; then
-    echo "$SCRIPT_NAME initializes the imagine-grinnell rails application"
+    echo "$SCRIPT_NAME initializes the ride-board rails application"
```

Comment:
Still a lie, the script lints, it doesn't initialize.

Reply:
I can totally read.